bigbluebutton-html5/imports/ui/components/webcam/component.jsx

```diff
@@ -236,9 +242,9 @@ const WebcamComponent = ({
               role="region"
               draggable={cameraDock.isDraggable && !isFullscreen ? 'true' : undefined}
               style={{
-                width: `${isDragging ? (cameraSize.width * 0.75) : (cameraDock.width * 0.75)}pt`,
-                height: `${isDragging ? (cameraSize.height * 0.75) : (cameraDock.height * 0.75)}pt`,
-                opacity: isDragging ? 0.5 : undefined,
+                width: isIphone ? mobileWidth : isDesktopWidth,
+                height: isIphone ? mobileHeight : isDesktopHeight,
+                opacity: camOpcity,
```

Comment:
```suggestion
                opacity: camOpcity,
```
Fix typo, see other comment